README.md

```diff
@@ -39,7 +39,7 @@ Rector can:
 ## Install
 
 ```bash
-composer require --dev rector/rector roave/better-reflection:dev-master#f8b3fc6
+composer require --dev rector/rector
```

Comment:
@TomasVotruba Shouldn't we suggest to install `dev-master` as our only release is out-of-date?